### PR TITLE
ci: fix macos build

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
None: try to fix MacOS CI

Describe changes:
- ci: fix macOS build

https://github.com/OISF/suricata/pull/10948 cleaner

### CODE REVIEW

**Review by jasonish — APPROVED** [review]
Looks good to get CI going again.

But is this something that most MacOS users will now need to do to build Suricata?
- Should Suricata auto-detect these paths better?

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -2589,8 +2590,8 @@ jobs:
       - run: tar xvf prep/libhtp.tar.gz
       - run: tar xvf prep/suricata-update.tar.gz
       - run: ./autogen.sh
-      - run: CFLAGS="${DEFAULT_CFLAGS}" ./configure --enable-unittests
-      - run: make -j ${{ env.CPUS }}
```
Comment (catenacyber) [inline .github/workflows/builds.yml:2593]: @jasonish I see that you committed this with 8522256aaa1

But env.CPUs does not seem to work here...

**Comment by catenacyber** [comment]
> But is this something that most MacOS users will now need to do to build Suricata?
> 
> * Should Suricata auto-detect these paths better?

There may be some magic spell to include these home-brew paths more natively

I am not sure we need all of them by the way, but now it is green :-p

File: .github/workflows/builds.yml (hunk 1 above)
Comment (jasonish) [reply .github/workflows/builds.yml:2593]: Yup, that was a mistake.

Comment (catenacyber) [reply .github/workflows/builds.yml:2593]: Is there more to fix ?

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #10971, thanks!

**Pull request closed without merging**